### ISSUE TITLE
Jar?

### ISSUE THREAD

**Issue by quintasoris** [wodxgod/Minecraft-Session-Token-Stealer#6]
Minecraft mods are .jar how will you get a python scrypt to run in the background

**Comment by ItsYoungDaddy** [comment]
do you even know what this does? cause i doubt you do

**Comment by quintasoris** [comment]
Yes it grabs the launcher_accounts file and sends it to a webhook so you can temporarily log in

**Comment by ExplicitPotato** [comment]
once you have the i.d, how do you log in?
